pkg/kubelet/prober/worker_test.go

```diff
@@ -58,18 +58,23 @@ func TestDoProbe(t *testing.T) {
 		failedStatus.Phase = v1.PodFailed
 
 		tests := []struct {
-			probe          v1.Probe
-			podStatus      *v1.PodStatus
-			expectContinue bool
-			expectSet      bool
-			expectedResult results.Result
+			probe                v1.Probe
+			podStatus            *v1.PodStatus
+			setDeletionTimestamp bool
+			expectContinue       bool
+			expectSet            bool
+			expectedResult       results.Result
 		}{
 			{ // No status.
 				expectContinue: true,
 			},
 			{ // Pod failed
 				podStatus: &failedStatus,
 			},
+			{ // Pod deletion
```

Comment:
I'm curious, why in terminated state test set looks like this:

```
{ // Container terminated
				podStatus:      &terminatedStatus,					podStatus:      &terminatedStatus,
				expectSet:      true,					expectSet:      true,
				expectedResult: results.Failure,					expectedResult: results.Failure,
			},				},
```

Should the terminated state be treated the same as being terminated?